simulation/lowfi_simulation/src/dev/native/cpp/main.cpp

```diff
@@ -0,0 +1,10 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include <HAL/HAL.h>
```

Comment:
This include isn't used and can be removed.

Reply:
I updated this to match what other "dev main" files have.  They use <iostream> which you have complained about, but it matches the format used throughout the rest of the project

Reply:
Which main.cpp files? The only ones I can find include HAL files only when they need them.